homeassistant/components/zha/api.py

```diff
@@ -285,13 +308,27 @@ async def websocket_get_group(hass, connection, msg):
     connection.send_result(msg[ID], group)
 
 
+def is_group_member(value: Any) -> GroupMember:
```

Comment:
hrm, by the name I'd expect it to return a bool. Should we use a different name, maybe `coerce_group_member` or `cv_group_member` as cv_ commonly used for  config validation?

Reply:
cv makes sense, I’ll update it

Reply:
done